receiver/fluentforwardreceiver/conversion.go

```diff
@@ -279,9 +282,6 @@ func (fe *ForwardEventLogRecords) DecodeMsg(dc *msgp.Reader) (err error) {
 
 		err = parseEntryToLogRecord(dc, lr)
 		if err != nil {
-			if msgp.Cause(err) == io.EOF {
```

Comment:
Is EOF now considered an actual error?

Reply:
Yes, because we know the number of records up front for this event type so any EOFs that happen in the middle of parsing indicate a truncated message.